javascript/builtins/intl/DateTimeFormat.json

```
@@ -629,7 +629,7 @@
                     "version_added": "3.0"
                   },
                   "webview_android": {
-                    "version_added": false
+                    "version_added": "4.4"
```

Did you test this one, too? If so, I'm concerned about the quality of pretty much all the Chromium-based browsers here. Chrome 35 doesn't line up with Android 4.4 (which is closer to but not exactly the same as Chrome 30, if I remember correctly) and doesn't line up with the (mutually inconsistent) Opera data.

If you tested this, then maybe the right thing to do here is to merge this change and open an issue saying this whole feature needs investigation.

---

Oooh, good catch! I did do a test for this, but I've looked at it again and it turned out to be a false positive by coincidence. I'm having trouble nailing down an accurate version for this one, and I'm not sure which version of `webview_android` is equivalent to Chrome desktop 35. What do you recommend I set the value to?

---

Let's go with ≤37. WebView versioning is a bit weird: it syncs with Chrome from 37, but before that, we don't know much about the various versions that were baked into Android. If Chrome 35 is accurate, this is good enough.

```suggestion
                    "version_added": "≤37"
```